### ISSUE TITLE
Auf1 is broken

### ISSUE THREAD

**Issue by BergbauerSepp** [yt-dlp/yt-dlp#8113]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting that yt-dlp is broken on a **supported** site
- [X] I've verified that I'm running yt-dlp version **2023.07.06** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [ ] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

austria

### Provide a description that is worded well enough to be understood

auf1 changed somthing

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
yt-dlp -vU https://auf1.tv/nachrichten-auf1/nachrichten-auf1-vom-14-september-2023
[debug] Command-line config: ['-vU', 'https://auf1.tv/nachrichten-auf1/nachrichten-auf1-vom-14-september-2023']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.07.06 [b532a3481] (pip)
[debug] Python 3.9.2 (CPython x86_64 64bit) - Linux-5.10.0-23-amd64-x86_64-with-glibc2.31 (OpenSSL 1.1.1n  15 Mar 2022, glibc 2.31)
[debug] exe versions: ffmpeg 4.3.6-0, ffprobe 4.3.6-0, phantomjs broken, rtmpdump 2.4
[debug] Optional libraries: Cryptodome-3.14.1, brotli-1.0.9, certifi-2020.06.20, mutagen-1.45.1, pyxattr-0.7.2, sqlite3-2.6.0, websocket                                                s-10.2
[debug] Proxy map: {}
[debug] Extractor Plugins: Auf1IE, Auf1RadioIE, DTubePluginIE, DTubeQueryPluginIE, DTubeSearchPluginIE, DTubeUserPluginIE, PmWissenIE, P                                                mWissenSearchIE, ServusSearchIE, ServusTVIE
[debug] Plugin directories: ['/usr/local/lib/python3.9/dist-packages/yt_dlp_plugins']
[debug] Loaded 1865 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Available version: stable@2023.07.06, Current version: stable@2023.07.06
yt-dlp is up to date (stable@2023.07.06)
[auf1] Extracting URL: https://auf1.tv/nachrichten-auf1/nachrichten-auf1-vom-14-september-2023
[auf1] nachrichten-auf1-vom-14-september-2023: Downloading JSON metadata
ERROR: [auf1] nachrichten-auf1-vom-14-september-2023: No video formats found!; please report this issue on  https://github.com/yt-dlp/yt                                                -dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/dist-packages/yt_dlp/YoutubeDL.py", line 1560, in wrapper
    return func(self, *args, **kwargs)
  File "/usr/local/lib/python3.9/dist-packages/yt_dlp/YoutubeDL.py", line 1709, in __extract_info
    return self.process_ie_result(ie_result, download, extra_info)
  File "/usr/local/lib/python3.9/dist-packages/yt_dlp/YoutubeDL.py", line 1768, in process_ie_result
    ie_result = self.process_video_result(ie_result, download=download)
  File "/usr/local/lib/python3.9/dist-packages/yt_dlp/YoutubeDL.py", line 2732, in process_video_result
    self.raise_no_formats(info_dict)
  File "/usr/local/lib/python3.9/dist-packages/yt_dlp/YoutubeDL.py", line 1073, in raise_no_formats
    raise ExtractorError(msg, video_id=info['id'], ie=info['extractor'],
yt_dlp.utils.ExtractorError: [auf1] nachrichten-auf1-vom-14-september-2023: No video formats found!; please report this issue on  https:                                                //github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -                                                U
```

**Comment by bashonly** [comment]
> [debug] Extractor Plugins: Auf1IE, Auf1RadioIE

We can't offer support for plugins. The plugin's dev(s) can do that

**Comment by BergbauerSepp** [comment]
[debug] Command-line config: ['-vU']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.07.06 [b532a3481] (pip)
[debug] Python 3.9.2 (CPython x86_64 64bit) - Linux-5.15.108-1-pve-x86_64-with-glibc2.33 (OpenSSL 1.1.1n  15 Mar 2022, glibc 2.33)
[debug] exe versions: ffmpeg 4.3.6-0, ffprobe 4.3.6-0, phantomjs broken, rtmpdump 2.4
[debug] Optional libraries: Cryptodome-3.11.0, brotli-1.0.9, certifi-2020.06.20, mutagen-1.45.1, pyxattr-0.7.2, sqlite3-2.6.0, websockets-10.1
[debug] Proxy map: {}
[debug] Extractor Plugins: Auf1IE, Auf1RadioIE, DTubePluginIE, DTubeQueryPluginIE, DTubeSearchPluginIE, DTubeUserPluginIE, PmWissenIE, PmWissenSearchIE, ServusSearchIE, ServusTVIE
[debug] Plugin directories: ['/usr/local/lib/python3.9/dist-packages/yt_dlp_plugins']
[debug] Loaded 1865 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Available version: stable@2023.07.06, Current version: stable@2023.07.06
yt-dlp is up to date (stable@2023.07.06)
